tannerb9/MasksNowTN-New
# tb-NewsUpdate

# Description
- Changed News from carousel to slide group.
- Moved the image in Info section within first p tag and floated it right so that text would wrap around it
- Implemented colors from logo to style site

## Type of change

# Testing Instructions
Run the app and check functionality of slide buttons on News section.
Review styling
